app/src/main/java/org/schabi/newpipe/util/DrawableResolver.kt

```diff
@@ -0,0 +1,26 @@
+package org.schabi.newpipe.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.annotation.AttrRes
+
+/**
+ * Utility class for resolving [Drawables](Drawable)
+ */
+class DrawableResolver {
```

Comment:
Regarding Kotlin, this should be an object class so that we don't need to use companion objects.